markets/perps-market/contracts/storage/SettlementStrategy.sol

```diff
@@ -26,6 +26,10 @@ library SettlementStrategy {
          * @dev the duration of the settlement window, after which committed orders can be cancelled.
          */
         uint256 settlementWindowDuration;
+        /**
+         * @dev the duration of the price window, after which price is not valid.
+         */
+        uint256 priceWindowDuration;
```

Comment:
We're not using setters for these? Is the idea that a new Strategy would be created if we want to update these?

Reply:
originally, yes. We can add later a module to update a settlement strategy if needed, but here and in spot, you can only create a new one and set the old as disabled.